OutOfSchool/OutOfSchool.WebApi.Tests/Controllers/ChildControllerTests.cs

```diff
@@ -92,6 +92,46 @@ public async Task GetChildById_WhenIdIsNotValid_ShouldReturnNull(long id)
             Assert.AreEqual(result.Value, null);
         }
 
+        [Test]
+        [TestCase(1)]
+        public async Task GetByParentId_WhenIdIsValid_ShouldReturnOkResultObject(long id)
+        {
+            // Arrange
+            service.Setup(x => x.GetAllByParent(id)).ReturnsAsync(children.Where(p => p.ParentId == id));
+
+            // Act
+            var result = await controller.GetByParentId(id).ConfigureAwait(false) as OkObjectResult;
+
+            // Assert
+            Assert.That(result, Is.Not.Null);
+            Assert.AreEqual(200, result.StatusCode);
```

Comment:
would be nice to use constants for 200 and 204 status code))
cold use this  Assert.IsNotNull(result); for isnot null check))

Reply:
Dear @yfedo,
Thank you a lot for your advice. I have implemented them in the current PR.